NumLib/IO/VtkIO/PVDFile.cpp

```diff
@@ -14,7 +14,9 @@
 #include <limits>
 #include <logog/include/logog.hpp>
 
-namespace FileIO
+namespace NumLib
```

Comment:
I'd put that file where also VTK output resides, because this class only makes sense at places where you do vtk output.